src/utils/calendar_manager.ts

```diff
@@ -961,6 +963,76 @@ const scheduleMeeting = async (
     throw error
   }
 }
+
+const updateGuestMeeting = async (
+  slotId: string,
+  startTime: Date,
+  endTime: Date,
+  participants: ParticipantInfo[],
+  meetingProvider: MeetingProvider,
+  meetingContent?: string,
+  meetingUrl?: string,
+  meetingTitle?: string,
+  meetingReminders?: Array<MeetingReminders>,
+  meetingRepeat = MeetingRepeat.NO_REPEAT,
+  selectedPermissions = [MeetingPermissions.SEE_GUEST_LIST]
+): Promise<MeetingDecrypted> => {
+  // Get the existing meeting to validate and get the meeting ID
+  const existingSlot = await getMeeting(slotId)
+
+  // Get the owner account to decrypt the meeting
+  const ownerAccount = await getAccount(existingSlot.account_address)
+
+  // For guest updates, we'll use the existing meeting ID from the slot
+  // We need to decrypt the existing meeting to get the meeting ID
+  const existingMeeting = await decryptMeeting(existingSlot, ownerAccount)
```

Comment:
This would fail as the guest does not have a valid signature to decrypt the meeting with.